collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/EncryptedFormTest.java

```diff
@@ -63,13 +63,13 @@ public void instanceOfEncryptedForm_cantBeViewedAfterSending() {
                 .swipeToEndScreen()
                 .clickSaveAndExit()
 
-                .clickSendFinalizedForm(1)
+                .clickSendFinalizedForm()
```

Comment:
I think we lose a potentially important assertion here. Could we keep the count argument in the `clickXForm` actions and just call `assertNumberOfXForms` in the implementation?

Reply:
There is a new method `assertNumberOfFinalizedForms` and it verifies the number but in other tests. I don't remember now but probably I assumed that we don't need to check the number every time we click that button (like here). There are tests that should check it and other tests like this one that doesn't have to. Do you think this should be done every time?

Reply:
I think for all existing tests we should check as that was coverage those tests were written with. We could go through and make a call for each one whether it's important or not, but seems easier just to either add a `assertNumberOfX` or keep the assertion in the `click` method.

Reply:
Ok done.